apps/framework-cli/src/framework/typescript/export_collectors.rs

```diff
@@ -21,21 +26,26 @@ pub enum ExportCollectorError {
     },
 }
 
-async fn collect_exports(file: &Path, project_path: &Path) -> Result<Value, ExportCollectorError> {
+async fn collect_exports(
+    command_name: &str,
+    process_name: &str,
+    file: &Path,
+    project_path: &Path,
+) -> Result<Value, ExportCollectorError> {
     let file_path_str = file.to_str().ok_or(ExportCollectorError::Other {
         message: "Did not get a proper file path to load exports from".to_string(),
     })?;
 
     let args = vec![file_path_str];
-    let process = bin::run(EXPORT_SERIALIZER_BIN, project_path, &args)?;
+    let process = bin::run(command_name, project_path, &args)?;
 
     let mut stdout = process
         .stdout
-        .expect("Data model config process did not have a handle to stdout");
+        .unwrap_or_else(|| panic!("{process_name} process did not have a handle to stdout"));
```

Comment:
Not  a big deal since that feels like exactly the change, why the change?

Reply:
Two process names, so the error message has to be dynamic. And clippy is not happy with `expect(format!(...))` (allocating a string even if no error)